addons/pharma/functions/fnc_getBloodVolumeChange.sqf

```diff
@@ -22,16 +22,13 @@ params ["_unit", "_deltaT", "_syncValues"];
 private _bloodLoss = [_unit] call ACEFUNC(medical_status,getBloodLoss);
 private _internalBleeding = GET_INTERNAL_BLEEDING(_unit);
 private _lossVolumeChange = (-_deltaT * ((_bloodLoss + _internalBleeding * (GET_HEART_RATE(_unit) / DEFAULT_HEART_RATE)) / GET_VASOCONSTRICTION(_unit)));
+private _enableFluidShift = EGVAR(vitals,enableFluidShift);
```

Comment:
why not just use EGVAR later?

Reply:
Because _enableFluidShift needs to be used both inside the actual fluid loss loop and the fluidShift loop at the bottom, so to keep it in scope for the fluidShift loop, it needs to be called early